src/check-project/files/.github/dependabot.yml

```
@@ -1,11 +1,27 @@
 version: 2
 updates:
 - package-ecosystem: npm
-  directory: "/"
+  directories:
+    - "/"
+    - "/packages/*"
```

Add other common folders too:

```suggestion
    - "/benchmarks/*"
    - "/docs/*"
    - "/examples/*"
    - "/interop/*"
    - "/packages/*"
```